src/navigationComponent.ts

```diff
@@ -128,18 +182,6 @@ export class NavigationComponent extends Component {
 					},
 				})
 		);
-
-		linkStates.sort((a, b) => {
```

Comment:
This code sorts the annotation links so that they appear in the order specified in the "Annotation strings" field in the settings. Removing this may change the current behavior.

Reply:
Oh yes, you're right! 

Now I've added this sorting back, and after checking the whole commits, I think the Windsurf has removed too much things.
I'll try to re-organize the all codes tomorrow  : )

Now this PR includes:
- add a "DevMode" switch in plugin settings
- support middle mouse click to open link in new tab
- support customizable property value as "property links"
  - and compatible with wiki links with display text
- add a setting to remove duplicated link
- add a setting to display the `title`(or other property) of target links